src/fileformats/ocd_file_export.cpp

```diff
@@ -1975,6 +1979,46 @@ void OcdFileExport::setupTextSymbolFraming(const TextSymbol* text_symbol, OcdTex
 }
 
 
+/**
+ * Returns true if the symbol may represent a line of secondary points.
+ */
+bool OcdFileExport::maybeSecondaryPoints(const Symbol* const symbol) const
+{
+	if (symbol->getType() != Symbol::Line)
+		return false;
+	
+	auto* line_symbol = static_cast<const LineSymbol*>(symbol);
+	if (line_symbol->getColor()
+	    || line_symbol->getLineWidth()
+	    || line_symbol->hasBorder())
+		return false;
+	
+	auto const is_empty = [line_symbol](auto getter) -> bool {
```

Comment:
Having `is_populated()` function here might help legibility. We are looking for a line symbol that has the mid-symbol populated and the other symbols empty. The test `!is_empty(...)` requires interpretation.